x-pack/plugins/ml/public/settings/_index.scss

```diff
@@ -1,3 +1,3 @@
 @import 'settings';
 @import 'filter_lists/index';
-@import 'scheduled_events/index';
\ No newline at end of file
+@import 'calendar/index';
```

Comment:
nit, we generally always add a new line to the end of the file.
In vscode there is a setting to auto add a new line on save.